src/GUI/Pages/CreateProject/CreateProjectModalWindow.razor

```diff
@@ -0,0 +1,40 @@
+@using LT.DigitalOffice.GUI.Services.ApiClients.ProjectService
+
+<div class="modal fade" id="exampleModalCenter" tabindex="-1" role="dialog" aria-labelledby="exampleModalCenterTitle" aria-hidden="true">
```

Comment:
Maybe it will better rename to kebab-case

Reply:
Fixed